java/registry/src/test/java/io/opensaber/registry/dao/impl/EncryptionDaoImplTest.java

```diff
@@ -49,13 +65,14 @@
 import io.opensaber.registry.middleware.util.Constants;
 import io.opensaber.registry.model.AuditRecordReader;
 import io.opensaber.registry.service.impl.EncryptionServiceImpl;
+import io.opensaber.registry.sink.DBProviderFactory;
 import io.opensaber.registry.sink.DatabaseProvider;
 import io.opensaber.registry.tests.utility.TestHelper;
 
 @Ignore
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = { RegistryDaoImpl.class, Environment.class, ObjectMapper.class, GenericConfiguration.class,
-		EncryptionServiceImpl.class, AuditRecordReader.class })
+		EncryptionServiceImpl.class })
```

Comment:
undo this and the Autowired commented out few lines below.

Reply:
Done.